src/pipelines/alphafold_inference_pipeline.py

```diff
@@ -94,38 +109,26 @@ def alphafold_inference_pipeline(
         loop_args=run_config.outputs['model_runners'], 
         parallelism=config.PARALLELISM
         ) as model_runner:
-    model_predict = PredictOp(
+    model_predict = JobPredictOp(
+        project=project,
+        location=region,
         model_features=data_pipeline.outputs['features'],
         model_params=model_parameters.output,
         model_name=model_runner.model_name,
         prediction_index=model_runner.prediction_index,
         run_multimer_system=run_config.outputs['run_multimer_system'],
         num_ensemble=run_config.outputs['num_ensemble'],
-        random_seed=model_runner.random_seed
+        random_seed=model_runner.random_seed,
+        tf_force_unified_memory=config.TF_FORCE_UNIFIED_MEMORY,
+        xla_python_client_mem_fraction=config.XLA_PYTHON_CLIENT_MEM_FRACTION
     ).set_display_name('Predict')
-    model_predict.set_cpu_limit(config.CPU_LIMIT)
-    model_predict.set_memory_limit(config.MEMORY_LIMIT)
-    model_predict.set_gpu_limit(config.GPU_LIMIT)
-    model_predict.add_node_selector_constraint(
-        config.GKE_ACCELERATOR_KEY, config.GPU_TYPE)
-    model_predict.set_env_variable(
```

Comment:
Why deleting these? It makes it not retro-compatible with the previous pipeline version, in the case there is no need to use the new environment variables and parameters.

Reply:
As per another the author reply on another change, `.set_env_variable()` doesn't work. Resolving this item.